documentation/asciidoc/accessories/audio/update-firmware.adoc

```diff
@@ -0,0 +1,45 @@
+== Updating your firmware
+
+Raspberry Pi Audio Boards use an EEPROM that contains information that is used by the host Raspberry Pi device to select the appropriate driver at boot time. This information is programmed into the EEPROM during manufacture. There are some circumstances where the end user may wish to update the EEPROM contents: this can be done from the command line.
+
+IMPORTANT: Before proceeding, you should update the Raspberry Pi OS running on your Raspberry Pi to the latest version.
+
+=== The EEPROM write-protect link
+
+During the programming process you will need to connect the two pads shown in the red box with a wire to pull down the EEPROM write-protect link.
+
+image::images/write_protect_tabs.jpg[width="80%"]
+
+NOTE: In some cases the two pads may already have a 0Ω resistor fitted to bridge the write-protect link, as illustrated in the picture of the Codec Zero board above.
+
+=== EEPROM Programming
+
+Once the write-protect line has been pulled down, the EEPROM can be programmed. 
+
+You should first install the utilites and then run the programmer. Open up a terminal window and type the following:
+
+----
+$ sudo apt install iqaudio-utils
+$ sudo iqaudio-program
```

Comment:
That should be
```
sudo apt update
sudo apt install rpi-audio-utils
sudo rpi-audio-flash
```

Reply:
Fixed in [e26881d](https://github.com/raspberrypi/documentation/pull/2767/commits/e26881da693fe896b79852c9a175cbe3a032fc93)

Reply:
I see that 'sudo apt update' was added, but the package and script names are still the `iqaudio` version.

Reply:
Oops.

Reply:
Fixed properly in [b57861e](https://github.com/raspberrypi/documentation/pull/2767/commits/b57861e6213820787f603724bb3e72e5aa205095)